tsconfig.json

```diff
@@ -7,6 +7,8 @@
     "lib": [
       "es6"
     ],
+    "inlineSourceMap": true,
```

Comment:
as stands, there is already a "sourceMap" option specified at line 5.   This one conflicts with it and generates a compile error on my machine